reflect_test.go

```diff
@@ -742,3 +744,40 @@ func TestPassthrough(t *testing.T) {
 		}
 	})
 }
+
+func Test_reflectNil(t *testing.T) {
+	tt(t, func() {
+		test, vm := test()
+		{
+			vm.Set("func", func(s *_mnoStruct) bool {
+				return s == nil
+			})
+
+			test(`
```

Comment:
If you look the starting ` (quote) is indented more than the closing one and the enclosing code is at the same level as the test method.

I guess go fmt won't touch it as its quoted text and hence not something it considers code?